src/condor_starter.V6.1/vanilla_proc.cpp

```diff
@@ -628,16 +615,16 @@ VanillaProc::StartJob()
 							return FALSE;
 						}
 						dprintf(D_FULLDEBUG, "Adding mapping: %s -> %s.\n", full_dir, next_dir);
```

Comment:
I think the `next_dir` needs to be `next_dir.c_str()`

Reply:
Done